doc/api/net.markdown

```diff
@@ -359,6 +359,10 @@ The amount of received bytes.
 
 The amount of bytes sent.
 
+### socket.destroyed
+
+A `boolean` value that depicts if the connection is destroyed or not. 
```

Comment:
I think "indicates" might be a better choice of word than "depicts". Maybe something like this:

> A `Boolean` value that indicates whether the connection has been destroyed or not.